dotnet-authserver/src/TeacherIdentity.AuthServer/AuthenticationState.cs

```diff
@@ -438,10 +442,11 @@ public void OnEmailChanged(string email)
         EmailAddress = email;
     }
 
-    public void OnNameSet(string? firstName, string? lastName)
+    public void OnNameSet(string? firstName, string? lastName, string? middleName = null)
```

Comment:
Nit: seems weird having `lastName` before `middleName` to me
```suggestion
    public void OnNameSet(string? firstName, string? middleName, string? lastName)
```

Reply:
Ah was just to avoid having to change every call to this method. Will change the order